core/src/main/scala/dagr/core/tasksystem/Task.scala

```diff
@@ -33,6 +33,12 @@ import scala.util.control.Breaks._
 /** Utility methods to aid in working with a task. */
 object Task {
 
+  /** A task that does nothing. */
+  def empty: Task = new Task {
```

Comment:
I think this is probably the right thing to do, but I'm also sort of curious if it would work with `val Empty = ..`, to have a single empty task.  My guess is it would probably work, but might be super confusing if the same empty task showed up at multiple very different places in a pipeline.

Reply:
It would work with it being a `val`, but just in case folks (including the `Pipeline` class) want to override the name or anything else, I don't think it's a good idea to have as a `val`.